<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkTextbookFormWindow</class>
 <widget class="QWidget" name="LinkTextbookFormWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="linkTextbook">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Link Textbook</string>
   </property>
  </widget>
  <widget class="QListView" name="allTextbooksListView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextbookLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All Textbooks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListView" name="registeredTextbooksListView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextbookLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Registered Textbooks</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
